### DISCUSSION TITLE
官方文档是不写错了？

### DISCUSSION THREAD

**Discussion by mengxh1990** [secretflow/secretflow#1365] [General]
这里几段伪代码怎么写的都是一样的，是写错了吗
https://www.secretflow.org.cn/zh-CN/docs/kuscia/main/reference/troubleshoot/kuscia_vs_ray
<img width="1862" alt="image" src="https://github.com/secretflow/secretflow/assets/12908155/51a4d1fe-20ed-4979-9cca-89dd221aac57">

**Comment by zimu-yuxi** [comment]
是的，我们会尽快修改
